pages/linux/sacct.md

```diff
@@ -0,0 +1,12 @@
+# sacct
+
+> Display accounting data from the Slurm service.
+> More information: <https://slurm.schedmd.com/sacct.html>.
+
+- Check basic items such as elapsed time, job name, cpus requested, and memory requested:
+
+`sacct -j {{job_id}} --format=elapsed,jobname,reqcpus,reqmem`
```

Comment:
We prefer the long argument format and the argument options should be between `{{}}`:
```suggestion
`sacct --jobs={{job_id}} --format={{elapsed}},{{jobname}},{{reqcpus}},{{reqmem}}`
```

Reply:
With `sacct` the `--format` option allows you to specify different fields you want to be displayed, in this case, elapsed, jobname, requested CPUs, and requested memory. I am double checking that the curly brackets around the fields would be correct in this situation, or would it be better to use:
`sacct --jobs={{job_id}} --format={{elapsed,jobname,reqcpus,reqmem}}`

 I have resolved the other comments in the latest commit.

Reply:
Checking the `tokei` page, is `{{}}` for each field.